geo/src/test/java/org/odk/collect/geo/support/FakeMapFragment.kt

```diff
@@ -82,9 +84,13 @@ class FakeMapFragment : Fragment(), MapFragment {
     }
 
     override fun addMarker(markerDescription: MarkerDescription): Int {
-        markers.add(markerDescription.point)
-        markerIcons.add(markerDescription.iconDescription)
-        return markers.size - 1
+        val featureId = generateFeatureId()
```

Comment:
We want to generate random IDs to prevent our tests from allowing implementations that rely on sequential ones.